src/status_im2/contexts/chat/photo_selector/view.cljs

```diff
@@ -100,14 +100,26 @@
         window-width    (:width (rn/get-window))]
     [:f>
      (fn []
-       (let [camera-roll-photos (rf/sub [:camera-roll/photos])
-             end-cursor         (rf/sub [:camera-roll/end-cursor])
-             loading?           (rf/sub [:camera-roll/loading-more])
-             has-next-page?     (rf/sub [:camera-roll/has-next-page])
-             selected-album     (or (rf/sub [:camera-roll/selected-album]) (i18n/label :t/recent))
-             blur-active?       (> @current-scroll min-scroll-to-blur)
-             window-height      (:height (rn/get-window))
-             top                (reanimated/use-shared-value window-height)]
+       (let [camera-roll-photos  (rf/sub [:camera-roll/photos])
+             end-cursor          (rf/sub [:camera-roll/end-cursor])
+             loading?            (rf/sub [:camera-roll/loading-more])
+             has-next-page?      (rf/sub [:camera-roll/has-next-page])
+             selected-album      (or (rf/sub [:camera-roll/selected-album]) (i18n/label :t/recent))
```

Comment:
A side comment about the `photo-selector` component. It's creating that functional component on the fly instead of being extracted to a separated `defn`. 
Do you think you can fix it in this PR?
If not, we should create a follow-up to clean this piece of code.

Reply:
@ulisesmac - I would suggest the refactoring in a separate PR/issue to keep the PR scope intact and resolve any bugs caused due to refactoring.